OrbitClientGgp/ClientGgp.h

```diff
@@ -8,6 +8,7 @@
 #include <cstdint>
 #include <memory>
 #include <string>
+#include <vector>
```

Comment:
Looks like `#include <vector>` is not needed

Reply:
I removed it, thanks!